conf/routes

```diff
@@ -105,6 +105,13 @@ POST   /v3/backfill/:name/start    @org.sagebionetworks.bridge.play.controllers.
 GET    /v3/cache           @org.sagebionetworks.bridge.play.controllers.CacheAdminController.listItems
 DELETE /v3/cache/:cacheKey @org.sagebionetworks.bridge.play.controllers.CacheAdminController.removeItem(cacheKey: String)
 
+# FOOTBALL PLAYERS HEATH STUDY API --------------------------------------------------------------------------
+
+GET    /fphs/externalId   @org.sagebionetworks.bridge.play.controllers.FPHSController.verifyExternalIdentifier(externalId: String ?= null)
```

Comment:
Minor: From a REST perspective, I feel like more of a verb action (verifying an existing externalId) vs a noun action (getting that externalId), so I feel like POST /fphs/externalId/verify?externalId=blah would be better. But I'll leave it to your judgement call.